Persephone/Controllers/WindowController.swift

```diff
@@ -20,6 +20,11 @@ class WindowController: NSWindowController {
     super.windowDidLoad()
     window?.titleVisibility = .hidden
 
+    NSEvent.addLocalMonitorForEvents(matching: .keyDown) {
+      self.keyDown(with: $0)
```

Comment:
Is the call to `self` necessary here?

Reply:
Xcode says yes.

"Call to method 'keyDown' in closure requires explicit 'self.' to make capture semantics explicit"